ht5.py

```diff
@@ -0,0 +1,48 @@
+class Human:
+    def __init__(self, name, sex, year_of_birth):
+        self.name = name
+        self.sex = sex
+        self.born = year_of_birth
+        self.age = int(time.ctime()[-4:]) - year_of_birth
```

Comment:
`time.ctime()` — 💪 
`time.ctime()[-4:]` — 🤨😐

Reply:
`time.strftime('%Y')`?
`datetime.datetime.now().year`?

https://stackoverflow.com/questions/30071886/how-to-get-current-time-in-python-and-break-up-into-year-month-day-hour-minu